metadata-service/openapi-servlet/src/test/resources/application.properties

```diff
@@ -0,0 +1,13 @@
+server.port=9090
```

Comment:
Don't like yml? 😛 Fine with it either way, but having it not consistent is a little strange.

Reply:
What do you mean?

Reply:
application.properties is the historical version of application.yml, both files get auto-pulled in by Spring Boot applications: https://www.baeldung.com/spring-boot-yaml-vs-properties

Reply:
Since we use application.yml everywhere else it feels a bit off to use properties here, but I'm not hard-line for yml

Reply:
I was trying to be consistent with the properties file on the `openapi-servlet` test resource.

Reply:
It looks like both the consumer jobs use application.properties anyway so we're already inconsistent. Ignore 😄